preferencesfx-demo/src/main/java/com/dlsc/preferencesfx/AppStarter.java

```diff
@@ -28,4 +30,15 @@ public void start(Stage primaryStage) {
     primaryStage.show();
     primaryStage.centerOnScreen();
   }
+
+  public AppStarter() {
+    addPreferencesListener();
```

Comment:
Great Idea! This makes it easier to see in the demo what is being done behind the scenes 👍